### ISSUE TITLE
0.3.1 fails: 'src/build.py' does not name an existing file

### ISSUE THREAD

**Issue by yurivict** [randy3k/rchitect#8]
```
===>  Configuring for py36-rchitect-0.3.1
error in rchitect setup command: 'src/build.py' does not name an existing file
*** Error code 1
```
tarball is from https://files.pythonhosted.org/packages/source/r/rchitect/rchitect-0.3.1.tar.gz

**Comment by randy3k** [comment]
Oh, you are right. I forgot to include the `src` folder in the tarball.

**Comment by randy3k** [comment]
fixed in v0.3.2
